connect-inject/container_init.go

```diff
@@ -37,6 +38,8 @@ type initContainerCommandData struct {
 	PrometheusScrapePath string
 	// PrometheusBackendPort configures where the listener on Envoy will point to.
 	PrometheusBackendPort string
+	// EnvoyUID is the UID that `consul connect redirect-traffic` will use when tproxy is enabled.
```

Comment:
```suggestion
	// EnvoyUID is the Linux user ID that `consul connect redirect-traffic` will use when tproxy is enabled.
```

For folks that don't know what UID stands for.